hedera-node/hedera-file-service-impl/src/test/java/com/hedera/node/app/service/file/impl/test/FileTestBase.java

```diff
@@ -77,12 +77,12 @@ public class FileTestBase {
     protected final FileID WELL_KNOWN_FILE_ID =
             FileID.newBuilder().fileNum(1_234L).build();
     protected final FileID WELL_KNOWN_UPGRADE_FILE_ID =
-            FileID.newBuilder().fileNum(150L).build();
+            FileID.newBuilder().fileNum(150L).shardNum(50L).realmNum(50L).build();
     protected final FileID WELL_KNOWN_SYSTEM_FILE_ID =
-            FileID.newBuilder().fileNum(150L).build();
+            FileID.newBuilder().fileNum(150L).shardNum(50L).realmNum(50L).build();
```

Comment:
System file should probably be different from upgrade file, perhaps 0.0.122

Reply:
I reverted it to 0.0.150

Reply:
It should be *different*, pick any reasonable non-upgrade file value.  We already have WELL_KNOWN_UPGRADE_FILE_ID, this should not be the same.

Reply:
fixed to 122